These are findings in the [golang/tools](http://github.com/golang/tools) repo, and they are all relevant: 
```
cmd/gotype/sizesFor19.go:5:1: [redundant-build-tag] The build tag "//go:build go1.9" is redundant for Go 1.25.0 and can be removed
go/analysis/internal/versiontest/version_test.go:5:1: [redundant-build-tag] The build tag "//go:build go1.21" is redundant for Go 1.25.0 and can be removed
cmd/godex/isAlias19.go:5:1: [redundant-build-tag] The build tag "//go:build go1.9" is redundant for Go 1.25.0 and can be removed
internal/typesinternal/varkind.go:5:1: [redundant-build-tag] The build tag "//go:build go1.25" is redundant for Go 1.25.0 and can be removed
go/internal/gccgoimporter/newInterface11.go:5:1: [redundant-build-tag] The build tag "//go:build go1.11" is redundant for Go 1.25.0 and can be removed
go/analysis/multichecker/multichecker_test.go:5:1: [redundant-build-tag] The build tag "//go:build go1.12" is redundant for Go 1.25.0 and can be removed
go/types/internal/play/play.go:5:1: [redundant-build-tag] The build tag "//go:build go1.23" is redundant for Go 1.25.0 and can be removed
go/callgraph/vta/internal/trie/trie_test.go:5:1: [redundant-build-tag] The build tag "//go:build go1.13" is redundant for Go 1.25.0 and can be removed
go/callgraph/vta/internal/trie/bits_test.go:5:1: [redundant-build-tag] The build tag "//go:build go1.13" is redundant for Go 1.25.0 and can be removed
go/analysis/passes/cgocall/cgocall_go121.go:5:1: [redundant-build-tag] The build tag "//go:build go1.21" is redundant for Go 1.25.0 and can be removed
go/ast/inspector/iter.go:5:1: [redundant-build-tag] The build tag "//go:build go1.23" is redundant for Go 1.25.0 and can be removed
```

--- a/RULES_DESCRIPTIONS.md
+++ b/RULES_DESCRIPTIONS.md
@@ -1414,7 +1414,7 @@ _Configuration_: N/A
 _Description_: This rule warns about redundant [build tag comments](https://pkg.go.dev/cmd/go@go1.17.0#hdr-Build_constraints).
 It detects unnecessary `// +build` comments when `//go:build` is present.
 `gofmt` in Go 1.17+ automatically adds the `//go:build` constraint, making the `// +build` comment unnecessary.
-Also, the rule spots redundant build tags `//go:build go1.X` when the package version is greater than or equal to `go1.X`.
+Also, the rule spots redundant build tags `//go:build go1.X` when the package's Go language version is greater than or equal to `go1.X`.
 
 ### Examples (redundant-build-tag)
 
@@ -1437,7 +1437,7 @@ After (fixed):
 package example
 ```
 
-Redundant build tag when the package version is Go 1.21 or later:
+Redundant build tag when the module Go version is Go 1.21 or later:
 
 Before (violation):
 

--- a/rule/redundant_build_tag.go
+++ b/rule/redundant_build_tag.go
@@ -21,7 +21,7 @@ func (*RedundantBuildTagRule) Apply(file *lint.File, _ lint.Arguments) []lint.Fa
 			if ver, ok := strings.CutPrefix(comment.Text, "//go:build "); ok {
 				hasGoBuild = true
 
-				// Starting from the Go 1.21, the go version in go.mod is a hard requirement.
+				// Starting with Go 1.21, the go version in go.mod is a hard requirement.
 				// Ignore this check for Go 1.20 and earlier.
 				if file.Pkg.IsAtLeastGoVersion(lint.Go121) && version.IsValid(ver) {
 					fileVersion := file.Pkg.GoVersion().String()

